### PR TITLE
feat(api): no longer exposing concept of links, remove corresponding …

### PR DESCRIPTION
**Pull request by Goncharo**
…enpdoints to internal

refs. metriport/metriport-internal#822

### Dependencies

- Downstream: https://github.com/metriport/metriport-internal/pull/832

### Description

- no longer exposing concept of links, remove corresponding enpdoints to internal
- also improve local testing DB init for dev account

### Release Plan

_[Document your release plan to minimize downtime]_

- [ ] merge this
- [ ] merge downstream

### CODE REVIEW

**Review by Orta21 — APPROVED** [review]
Looks good, just maybe remove the gitHeads and im assuming the other package upgrades was on purpose

File: docs/medical-api/more-info/faq.mdx
Comment (leite08) [inline docs/medical-api/more-info/faq.mdx:1]: ditto: why removed

File: packages/packages/api/package.json [hunk 1]
```diff
@@ -49,5 +49,6 @@
     "prettier": "^2.8.3",
     "ts-essentials": "^9.3.1",
     "typescript": "^4.9.5"
-  }
+  },
+  "gitHead": "8f867432188b03c93f8d9bc86ce090fbcd8db5e2"
```
Comment (leite08) [inline packages/packages/api/package.json:53]: left-over

File: packages/packages/api/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metriport/api",
-  "version": "4.1.10",
+  "version": "4.1.11-alpha.2",
```
Comment (leite08) [inline packages/packages/api/package.json:3]: We're introducing breaking changes, this is a major change

File: packages/packages/commonwell-cert-runner/package.json [hunk 3]
```diff
@@ -60,5 +60,6 @@
     "nodemon": "^2.0.20",
     "ts-node": "^10.9.1",
     "typescript": "^4.9.5"
-  }
+  },
+  "gitHead": "8f867432188b03c93f8d9bc86ce090fbcd8db5e2"
```
Comment (leite08) [inline packages/packages/commonwell-cert-runner/package.json:64]: left-over

File: packages/packages/commonwell-jwt-maker/package.json [hunk 4]
```diff
@@ -52,5 +52,6 @@
     "nodemon": "^2.0.20",
     "ts-node": "^10.9.1",
     "typescript": "^4.9.5"
-  }
+  },
+  "gitHead": "8f867432188b03c93f8d9bc86ce090fbcd8db5e2"
```
Comment (leite08) [inline packages/packages/commonwell-jwt-maker/package.json:56]: left-over

File: packages/packages/commonwell-sdk/package.json [hunk 5]
```diff
@@ -50,5 +50,6 @@
     "eslint-config-prettier": "^8.6.0",
     "prettier": "^2.8.3",
     "typescript": "^4.9.5"
-  }
+  },
+  "gitHead": "8f867432188b03c93f8d9bc86ce090fbcd8db5e2"
```
Comment (leite08) [inline packages/packages/commonwell-sdk/package.json:54]: left-over

File: packages/packages/react-native-sdk/package.json [hunk 6]
```diff
@@ -68,5 +68,6 @@
   },
   "dependencies": {
     "react-native-webview": "^11.26.1"
-  }
+  },
+  "gitHead": "8f867432188b03c93f8d9bc86ce090fbcd8db5e2"
```
Comment (leite08) [inline packages/packages/react-native-sdk/package.json:72]: left-over

File: api/app/docker-compose.dev.yml [hunk 7]
```diff
@@ -28,6 +28,8 @@ services:
       - TOKEN_TABLE_NAME=token
       - DYNAMODB_ENDPOINT=http://dynamodb:8000
       - FHIR_CONVERTER_SERVER_URL=http://host.docker.internal:8777
+      - MEDICAL_DOCUMENTS_BUCKET_NAME=medical-documents-local
+      - LOCAL_ACCOUNT_CXID=cdb678ab-07e3-42c5-93f5-5541cf1f15a8
```
[the remark this answers is not in the document]
Comment (Goncharo) [reply api/app/docker-compose.dev.yml:32]: fixed

File: docs/devices-api/more-info/faq.mdx
[the remark this answers is not in the document]
Comment (Goncharo) [reply docs/devices-api/more-info/faq.mdx:1]: it's being indexed by mintlify when we don't want it to (not part of mint.json either)

File: docs/medical-api/more-info/faq.mdx
Comment (Goncharo) [reply docs/medical-api/more-info/faq.mdx:1]: https://github.com/metriport/metriport/pull/610#discussion_r1240963702

File: api/app/docker-compose.dev.yml (hunk 7 above)
[the remark this answers is not in the document]
Comment (leite08) [reply api/app/docker-compose.dev.yml:32]: the ID's still there, maybe missed pushing the changes to remote? :)

**Pull request merged**